doc: fix structure issues in some libs + minor doc fixes (#7403)
nrfconnect/sdk-nrf: doc/nrf/libraries/modem/modem_jwt.rst

The Modem JWT library provides access to the modem's `JSON Web Token (JWT)`_ generation feature, which is available in modem firmware v1.3.0 or higher.

Configuration
*************

To use the library to request a JWT, complete the following steps:

> 1. Enable the library by setting the following Kconfig options:

Reviewer: ```suggestion
1. Set the following Kconfig options to enable the library:
```
I always want to desperately avoid the structure "Do something by ...ing something." Up to you, of course.
